finetuner/__init__.py

```diff
@@ -206,6 +209,19 @@ def fit(
         in a batch. For example, if ``batch_size`` is 20, and
         ``num_items_per_class`` is 4, the batch will consist of 4 items for each of
         the 5 classes. Batch size must be divisible by `num_items_per_class`.
+    :param sampler: Determines which sampling method will be used in the case that the
+        data is labeled. By default is `auto`, meaning that the appropriate sampler,
+        will be chosen depending on the loss function used. if set to `random` then
+        `num_items_per_class` is not used.
```

Comment:
I think it would be good to add the 3 possible options here as for the other parameters.